### PR TITLE
modifyFunction

### LINKED ISSUE #10: Update the UG
Move the draft UG content into the User Guide page

### LINKED ISSUE #35: Update UG Yong Ning

### PR DESCRIPTION
**Pull request by yongning0310**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -96,6 +96,21 @@ Shows a list of all internships in the address book.
 
 Format: `list`
 
+### Editing an internship : `modify`
+
+Edits an existing internship in the internship list.
+
+Format: `modify INDEX c/COMPANY_NAME ro/ROLE a/APPLICATION_STATUS s/START_DATE d/DURATION [re/REQUIREMENT]...`
```
Comment (shashahchk) [inline docs/UserGuide.md]: is the [re/REQUIREMENT] a typo?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 # AB-3 User Guide
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+flagShip is a **desktop app for managing internships, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, flagShip can get your internship management tasks done faster than traditional GUI apps.
```
Comment (shashahchk) [inline docs/UserGuide.md]: possibly capitalized?

**Review by pygmychameleon — COMMENTED** [review]
LGTM. a few nits

**Review by pygmychameleon — APPROVED** [review]
LGTM

**Pull request merged**